hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestListKeys.java

```diff
@@ -369,32 +371,22 @@ private void checkKeyShallowList(String keyPrefix, String startKey,
     assertEquals(keys, outputKeysList);
   }
 
-  private static void createKeys(OzoneBucket ozoneBucket, List<String> keys)
+  private static void createAndAssertKeys(OzoneBucket ozoneBucket, List<String> keys)
       throws Exception {
     int length = 10;
     byte[] input = new byte[length];
     Arrays.fill(input, (byte) 96);
     for (String key : keys) {
-      createKey(ozoneBucket, key, 10, input);
-    }
-  }
-
-  private static void createKey(OzoneBucket ozoneBucket, String key, int length,
-      byte[] input) throws Exception {
+      createKey(ozoneBucket, key, ReplicationFactor.THREE, ReplicationType.RATIS,
+          new String(input, StandardCharsets.UTF_8));
 
-    OzoneOutputStream ozoneOutputStream =
-        ozoneBucket.createKey(key, length);
+      // Read the key with given key name.
+      OzoneInputStream ozoneInputStream = ozoneBucket.readKey(key);
+      byte[] read = new byte[length];
+      ozoneInputStream.read(read, 0, length);
+      ozoneInputStream.close();
 
-    ozoneOutputStream.write(input);
-    ozoneOutputStream.write(input, 0, 10);
-    ozoneOutputStream.close();
-
-    // Read the key with given key name.
-    OzoneInputStream ozoneInputStream = ozoneBucket.readKey(key);
-    byte[] read = new byte[length];
-    ozoneInputStream.read(read, 0, length);
-    ozoneInputStream.close();
-
-    assertEquals(new String(input, StandardCharsets.UTF_8), new String(read, StandardCharsets.UTF_8));
+      assertEquals(new String(input, StandardCharsets.UTF_8), new String(read, StandardCharsets.UTF_8));
+    }
```

Comment:
Let's keep this method as `readKey` without the write part.  So the loop would look like:

```java
  for (String key : keys) {
    createKey(...)
    readKey(...)
  }
```